src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
As i understand it's set of metrics, do we need it now?

Reply:
It s a part of official create-react-app. I'm not sure we need it now or not.